packages/airnode-node/src/evm/fulfillments/api-calls.test.ts

```diff
@@ -246,7 +245,7 @@ describe('submitApiCall', () => {
         expect(data).toEqual({
           ...apiCall,
           fulfillment: { hash: '0xfailtransaction' },
-          status: RequestStatus.Submitted,
+          status: RequestStatus.Errored,
```

Comment:
Why is this errored? Is it because we were rewriting the status in fulfillments/requests.ts?

Reply:
Yes, as you said this was a small bug- the test is submitting a fail transaction so `Errored` is appropriate.